### PR TITLE
CNW rebranding

### PR DESCRIPTION
**Pull request by idradm**

### CODE REVIEW

File: extensions/wikia/CreateNewWiki/js/WikiBuilder.js [hunk 1]
```diff
@@ -572,15 +627,16 @@ define('ext.createNewWiki.builder', ['ext.createNewWiki.helper', 'wikia.tracker'
...
+						wVertical: wikiVertical.val(),
 						wCategories: categories,
 						wAllAges: wikiAllAges.is(':checked') ? wikiAllAges.val() : null
 					},
 					token: preferencesToken
 				},
 				callback: function (res) {
 					throbberWrapper.stopThrobbing();
+					throbberWrapper.removeClass('creating-wiki');
 					cityId = res.cityId;
```
Comment (WikiaTech) [inline extensions/wikia/CreateNewWiki/js/WikiBuilder.js:640]: MAJOR Remove this useless assignment to local variable "cityId" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)

File: extensions/wikia/CreateNewWiki/js/WikiBuilder.js [hunk 2]
```diff
@@ -572,15 +627,16 @@ define('ext.createNewWiki.builder', ['ext.createNewWiki.helper', 'wikia.tracker'
...
 					createStatus = res.status;
 					createStatusMessage = res.statusMsg;
```
Comment (WikiaTech) [inline extensions/wikia/CreateNewWiki/js/WikiBuilder.js:642]: MAJOR Remove this useless assignment to local variable "createStatusMessage" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)

File: extensions/wikia/CreateNewWiki/js/WikiBuilder.js [hunk 3]
```diff
@@ -78,18 +83,19 @@ define('ext.createNewWiki.builder', ['ext.createNewWiki.helper', 'wikia.tracker'
...
-		wikiLanguage = $nameWikiWrapper.find('select[name=wiki-language]');
-		wikiVertical = $descWikiWrapper.find('select[name=wiki-vertical]');
+		wikiLanguage = $nameWikiWrapper.find('input[name=wiki-language]');
+		wikiLanguageList = $nameWikiWrapper.find('.wiki-language-dropdown');
+		wikiVertical = $descWikiWrapper.find('input[name=wiki-vertical]');
+		wikiVerticalList = $descWikiWrapper.find('.wiki-vertical-dropdown');
+		wikiVerticalError = $descWikiWrapper.find('.wiki-vertical-error');
 		wikiAllAges = $descWikiWrapper.find('input[name=all-ages]');
 		allAgesDiv = $('#all-ages-div');
 		descWikiSubmitError = $descWikiWrapper.find('.submit-error');
```
Comment (WikiaTech) [inline extensions/wikia/CreateNewWiki/js/WikiBuilder.js:101]: MAJOR Remove this useless assignment to local variable "descWikiSubmitError" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 6 issues
* MAJOR 6 major

Watch the comments in this conversation to review them.

#### 3 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [WikiBuilder.js#L9](https://github.com/Wikia/app/blob/e5973165a5348b834bb5cfa0937f9ec9aadcd762/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L9): Remove this useless assignment to local variable "createStatusMessage" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
1. MAJOR [WikiBuilder.js#L10](https://github.com/Wikia/app/blob/e5973165a5348b834bb5cfa0937f9ec9aadcd762/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L10): Remove this useless assignment to local variable "cityId" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
1. MAJOR [WikiBuilder.js#L502](https://github.com/Wikia/app/blob/e5973165a5348b834bb5cfa0937f9ec9aadcd762/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L502): Reduce the number of conditional operators (5) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 6 issues
* MAJOR 6 major

Watch the comments in this conversation to review them.

#### 3 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [WikiBuilder.js#L9](https://github.com/Wikia/app/blob/00812654468b2b3cff4c07c85207d3eeab75c6a2/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L9): Remove this useless assignment to local variable "createStatusMessage" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
1. MAJOR [WikiBuilder.js#L10](https://github.com/Wikia/app/blob/00812654468b2b3cff4c07c85207d3eeab75c6a2/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L10): Remove this useless assignment to local variable "cityId" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
1. MAJOR [WikiBuilder.js#L502](https://github.com/Wikia/app/blob/00812654468b2b3cff4c07c85207d3eeab75c6a2/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L502): Reduce the number of conditional operators (5) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 6 issues
* MAJOR 6 major

Watch the comments in this conversation to review them.

#### 3 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [WikiBuilder.js#L9](https://github.com/Wikia/app/blob/0b5968c56fa61f64abc63983887bb805b815362b/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L9): Remove this useless assignment to local variable "createStatusMessage" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
1. MAJOR [WikiBuilder.js#L10](https://github.com/Wikia/app/blob/0b5968c56fa61f64abc63983887bb805b815362b/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L10): Remove this useless assignment to local variable "cityId" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
1. MAJOR [WikiBuilder.js#L502](https://github.com/Wikia/app/blob/0b5968c56fa61f64abc63983887bb805b815362b/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L502): Reduce the number of conditional operators (5) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 6 issues
- MAJOR 6 major

Watch the comments in this conversation to review them.

#### 3 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:
1. MAJOR [WikiBuilder.js#L9](https://github.com/Wikia/app/blob/5ff137dd53441282cea53c8c167cfe6cee7dc86f/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L9): Remove this useless assignment to local variable "createStatusMessage" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
2. MAJOR [WikiBuilder.js#L10](https://github.com/Wikia/app/blob/5ff137dd53441282cea53c8c167cfe6cee7dc86f/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L10): Remove this useless assignment to local variable "cityId" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
3. MAJOR [WikiBuilder.js#L502](https://github.com/Wikia/app/blob/5ff137dd53441282cea53c8c167cfe6cee7dc86f/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L502): Reduce the number of conditional operators (5) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)

**Review by lukaszkonieczny — APPROVED** [review]

**Pull request merged**